test/smoke/Makefile.rules

```diff
@@ -25,7 +29,11 @@ else ifdef march
 else ifdef make_options
 	$(CC) $(make_options) -o $@
 else
+ifneq (,$(findstring $(INSTALLED_GPU),$(SUPPORTED)))
```

Comment:
and to add UNSUPPORTED functionality, I think you just need a nested:

ifeq (,$(findstring $(INSTALLED_GPU),$(UNSUPPORTED)))

and another else echo Skipped due to unsupported